bookkeeper-server/src/main/proto/DataFormats.proto

```diff
@@ -58,6 +58,15 @@ message LedgerMetadataFormat {
         optional bytes value = 2;
     }
     repeated cMetadataMapEntry customMetadata = 11;
+
+    /**
+    * Ledger Type
+    */
+    enum LedgerType {
```

Comment:
This is the third definition of the same enum.

Reply:
I would like to use an integer or to use the same type